js/ui/components/clubhouse.js

```diff
@@ -577,6 +611,83 @@ class ClubhouseQuestBanner extends ClubhouseNotificationBanner {
             });
         }
     }
+
+    setBottomBanner(bottomBanner) {
+        this._bottomBanner = bottomBanner;
+    }
+
+    _updateUpDownIcon() {
+        if (!this._upDownIcon)
+            return;
+
+        if (this._position.atBottom)
+            this._upDownIcon.icon_name = 'go-top-symbolic';
+        else
+            this._upDownIcon.icon_name = 'go-bottom-symbolic';
+    }
+
+    _setupUpDownButton() {
+        this._upDownIcon = new St.Icon({ icon_name: 'go-bottom-symbolic',
+                                         icon_size: 16 });
```

Comment:
Nitpick: 

```
this._upDownIcon = new St.Icon({
    icon_name: 'go-bottom-symbolic',
    icon_size: 16,
});
```

Reply:
Good catch @GeorgesStavracas . I just updated the PR.